src/MyPlot/MyPlot.php

```diff
@@ -414,14 +414,7 @@ public function getPlotMid(Plot $plot) {
 
 		$plotSize = $plotLevel->plotSize;
 		$pos = $this->getPlotPosition($plot);
-		if($plot->X >= 0 and $plot->Z >= 0)
-			$pos->add(floor($plotSize / 2), 1, floor($plotSize / 2));
-		if($plot->X < 0 and $plot->Z > 0)
-			$pos->add(-floor($plotSize / 2), 1, floor($plotSize / 2));
-		if($plot->X > 0 and $plot->Z < 0)
-			$pos->add(floor($plotSize / 2), 1, -floor($plotSize / 2));
-		if($plot->X < 0 and $plot->Z < 0)
-			$pos->add(-floor($plotSize / 2), 1, -floor($plotSize / 2));
+		$pos = new Position($pos->getFloorX() + ($plotSize / 2) + 0.5, $pos->getFloorY() + 1, $pos->getFloorZ() + ($plotSize / 2) + 0.5);
```

Comment:
![image](https://user-images.githubusercontent.com/8733998/29691829-17bf6326-892d-11e7-891d-07270eba4a85.png)
Maybe this image explains it. the position of the plot is always at the lowest x/z